### PR TITLE
53 implement a form field for the user to input starting values into the starting state

### LINKED ISSUE #53: Implement a form field for the user to input starting values into the starting state.
Points: 5
Edit the user interface in order for the user to click a button and add input starting values

### PR DESCRIPTION
**Pull request by Fahim-A-Karim**
Just simple fields to html file and tests

### CODE REVIEW

File: logicsim/templates/logicsim/index.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <p>
         <h1>Logic Gate Simulator</h1>
         <ol>
-            {% for gate in logicsim %}
+            {% for gate in db %}
```
Comment (Nguyen-Tommy) [inline logicsim/templates/logicsim/index.html]: This should not be changed

File: logicsim/templates/logicsim/UserInput.html [hunk 2]
```diff
@@ -22,6 +22,10 @@
   <label for="input2">Input 2: </label>
   <input type="number" id="input2" name="input2"><br><br>
 
+  <label for="SS1">Start State 1: </label>
```
Comment (Nguyen-Tommy) [inline logicsim/templates/logicsim/UserInput.html:25]: This section of code should be in index.html.
.

**Review by GAliaWeiss — APPROVED** [review]
A starting state was added to the user input form with a range of acceptable values included.  Syntax on line 30 was corrected.

**Review by Nguyen-Tommy — CHANGES_REQUESTED** [review]

File: logicsim/templates/logicsim/index.html (hunk 1 above)
Comment (Fahim-A-Karim) [reply logicsim/templates/logicsim/index.html]: fixed

File: logicsim/templates/logicsim/UserInput.html (hunk 2 above)
Comment (Fahim-A-Karim) [reply logicsim/templates/logicsim/UserInput.html:25]: fixed

**Review by Nguyen-Tommy — APPROVED** [review]

**Pull request merged**